packages/aila/src/lib/agentic-system/agents/sectionAgents/subjectAgent/subjectsByKeyStage.ts

```diff
@@ -0,0 +1,119 @@
+export const subjectsByKeyStage: Record<string, string[]> = {
```

Comment:
This could drift from the truth, but I don't have a great process suggestion to make sure it stays up to date